### DISCUSSION TITLE
'airflow.models.taskinstance.TaskInstance object' has no attribute 'map_index'

### DISCUSSION THREAD

**Discussion by ImSagarbhatt** [apache/airflow#26484] [Q&A]
I am trying to download the logs but getting the following error
File "/home/ubuntu/airflow/env/lib/python3.8/site-packages/jinja2/_compat.py", line 28, in reraise raise value.with_traceback(tb)
File "<template>", line 1, in top-level template code
jinja2.exceptions.UndefinedError: 'airflow.models.taskinstance.TaskInstance object' has no attribute 'map_index'.

Please help me out here.
Thanks in advance.

**Comment by potiuk** [comment]
Help from my side: There is an error. Until you provide enough details. there is no way anyone can help you, I m afraid.

Try to make sure your airflow is installed properly (with constraints - see https://airflow.apache.org/docs/apache-airflow/stable/installation/installing-from-pypi.html and has the right database and the database is upgraded: https://airflow.apache.org/docs/apache-airflow/stable/installation/upgrading.html
